### PR TITLE
Removed dependency on Lodash

### PR DESCRIPTION
**Pull request by kunagpal**

### CODE REVIEW

**Comment by shamasis** [comment]
Also change log.

**Comment by kunagpal** [comment]
@shamasis Done.

**Comment by kunagpal** [comment]
@shamasis Ping

**Comment by shamasis** [comment]
@kunagpal this is a major version bump

File: README.md [hunk 1]
```diff
@@ -7,3 +7,46 @@
...
+```
+
+Check the [tests](https://github.com/postmanlabs/chai-postman/tree/develop/test/unit) for a complete reference.
+
+## Compatibility
+
+| chai-postman | postman-collection |  lodash |
+|:------------:|:------------------:|:-------:|
+|    v0.1.0    |        v3.x        | v4.17.4 |
+|    v0.1.1    |        v3.x        | v4.17.4 |
```
Comment (kamalaknn) [inline README.md]: Needs a trailing newline.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
 # Chai Postman Changelog
 
+#### v0.1.1 [Unreleased] (November 4, 2017)
```
Comment (kamalaknn) [inline CHANGELOG.md]: Unreleased versions should not have dates.

Comment (kamalaknn) [reply CHANGELOG.md]: I think this is a breaking change. Can we make this `0.2.0` at least?

`1.0.0` would be best.

Comment (shamasis) [reply CHANGELOG.md]: @kunagpal - yes - remove this 0.0.1

```
#### Unreleased
- ...
- ...
- ...
```

**Pull request merged**